src/NexusMods.Abstractions.Games.FileHashes/Models/VersionDefinition.cs

```diff
@@ -38,6 +39,11 @@ public partial class VersionDefinition : IModelDefinition
     /// </summary>
     public static readonly StringsAttribute Steam = new(Namespace, nameof(Steam)) { IsIndexed = true };
     
+    /// <summary>
+    /// The associated Steam ManifestIDs for this version
```

Comment:
I think it's supposed to be
```suggestion
    /// The associated Epic ManifestIDs for this version
```